bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/wrappers/veneer/TestAdminClientVeneerApi.java

```diff
@@ -144,6 +163,61 @@ public void dropAllRowsAsync() throws Exception {
     assertEquals(tableId, NameUtil.extractTableIdFromTableName(rangeRequest.getName()));
   }
 
+  @Test
+  public void createBackupAsync() throws ExecutionException, InterruptedException {
+    com.google.cloud.bigtable.admin.v2.models.CreateBackupRequest request =
+        com.google.cloud.bigtable.admin.v2.models.CreateBackupRequest.of(CLUSTER_ID, BACKUP_ID)
+            .setSourceTableId(TABLE_NAME);
+
+    adminClientWrapper.createBackupAsync(request).get();
+    com.google.bigtable.admin.v2.CreateBackupRequest backupRequest =
+        fakeAdminService.popLastRequest();
+    assertEquals(
+        "projects/fake-project-id/instances/fake-instance-id/tables/projects/fake-project-id/instances/fake-instance-id/tables/fake-Table-id-1",
+        backupRequest.getBackup().getSourceTable());
+  }
+
+  @Test
+  public void listBackupsAsync() throws ExecutionException, InterruptedException {
+    List<String> listApiFuture = adminClientWrapper.listBackupsAsync(CLUSTER_ID).get();
+    assertEquals(1, listApiFuture.size());
+    assertEquals("fake-backup-id", listApiFuture.get(0));
+  }
+
+  @Test
+  public void testDeleteBackupAsync() throws Exception {
+    Future<Void> deleteResponse = adminClientWrapper.deleteBackupAsync(CLUSTER_ID, BACKUP_ID);
+    deleteResponse.get();
+    DeleteBackupRequest deleteBackupRequest = fakeAdminService.popLastRequest();
+    assertEquals(
+        "projects/fake-project-id/instances/fake-instance-id/clusters/fake-cluster-id/backups/fake-backup-id",
+        deleteBackupRequest.getName());
+  }
+
+  private static class FakeOperationsGrpc extends OperationsGrpc.OperationsImplBase {
+    @Override
+    public void getOperation(
+        GetOperationRequest request, StreamObserver<Operation> responseObserver) {
+      if (callCount.getAndIncrement() < 3) {
+        responseObserver.onError(Status.DEADLINE_EXCEEDED.asException());
```

Comment:
I think it would be more appropriate to test polling behavior by returning OK(done = false) instead of an error which tests retry behavior

Reply:
done